analyzers/tests/SonarAnalyzer.UnitTest/TestCases/InsteadOfAny.EntityFramework.cs

```diff
@@ -17,12 +17,11 @@ public class MyDbContext : DbContext
 
     public void GetEntities(MyDbContext dbContext, List<int> ids)
     {
-        _ = dbContext.MyEntities.Where(e => ids.Any(i => e.Id == i)); // Noncompliant - FP, should raise in context of EntityFramework queries. Exist cannot be translated to SQL query
+        _ = dbContext.MyEntities.Where(e => ids.Any(i => e.Id == i)); // Compliant
+        _ = dbContext.MyEntities.Where(e => ids.Any(i => e.Equals(i))); // Compliant
+        _ = dbContext.MyEntities.Where(e => ids.Any(i => e.Id > i)); // Compliant
+
         _ = dbContext.MyEntities.Where(e => ids.Any(i => e.Id is i)); // Error [CS0150]
-        // Noncompliant@+1
         _ = dbContext.MyEntities.Where(e => ids.Any(i => e.Id is 2)); // Error [CS8122]
-        _ = dbContext.MyEntities.Where(e => ids.Any(i => e.Equals(i))); // Noncompliant - FP, should raise in context of EntityFramework queries. Exist cannot be translated to SQL query
```

Comment:
why did you remove this?

Reply:
I did not remove it, I just moved all the Noncm together and the errors after them